packages/ui-karma-config/lib/index.js

```diff
@@ -197,6 +197,14 @@ module.exports = function makeConfig({ coverageDirectory, coverageThreshold }) {
 
       webpackMiddleware: { stats: 'errors-only' },
 
+      client: {
+        mocha: {
+          // change the default test timeout to 5 secs. This must be higher
+          // than our async test timeouts (currently 1.9 secs) because if this
+          // happens first we do not get any sensible error message.
+          timeout: 5000
+        }
+      },
```

Comment:
this is a small fix, that makes our tests return error messages when the `await find...` calls in tests do not return in time. Their timeout was set to 1900ms, which Mocha's default is 2000ms. And when Mocha timeouts first it returns no details about was timed out.